docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
+
+`Orientation` and `Scale` of `View` are also affected by the transforms of its parent. In the final calculation of `View`, `Orientation` and `Scale` of parents are first applied to the child `View` with parent's `PivotPoint`, and then `Orientation` and `Scale` of child are applied.
+
+The following examples demonstrate the rotation and scaling that change `Orientation` and `Scale` respectively. `View` **A** is a child of `View` **B**. Therefore, transforms of **A** only affects **A**. however, the transforms of parent **B** influences its child **A**. The black dot in the figure is `PivotPoint` that is used by the transform.
+
+**Figure: Rotation and Scaling.**
+
+<table style="width:100%">
+<tr>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Orientation.gif" width="100%">
+</td>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Scale.gif" width="100%">
+</td>
+</tr>
+</table>
 
-```
-TextLabel label = new TextLabel( "Hello World" );
-label.BackgroundColor = Color.Red;
-```
 
-**Figure: TextLabel object with a red background**
+<a name="2"></a>
+## Directional Navigation
+
+NUI also provides directional navigation between `View` using arrow keys on the keyboard. You can simply set the following properties to specify the next `FocusableViews` for each direction:
+
+- [View.UpFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_UpFocusableView)
+
+- [View.DownFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_DownFocusableView)
+
+- [View.LeftFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_LeftFocusableView)
+
+- [View.RightFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_RightFocusableView)
+
+When an arrow key is pressed and the current `View` has next `FocusableView` for that direction, the system passes focus to the next `FocusableView`. However, if the current `View` does not have the next `FocusableView` for the direction or the next `ForcusableView` is not on the window, the property returns `NULL` and no change occurs.
 
-![TextLabel object with a red background](./media/background_textlabel.png)
 
 <a name="3"></a>
-## Setting the Background Image
+## View Event
 
-You can set a background image of a view:
+`View` provides a variety of `events` that are commonly used by the components. The following table lists the basic events provided by `View`:
 
-```
-View view = new View();
-view.BackgroundImage = "image.png";
+**Table: View events**
+
+| Event               | Description                                               |
+| ------------------- | --------------------------------------------------------- |
```

Comment:
Its good suggestion but I think we need to change most of the sentences to look better.

@mijin85cho and @Reni-90, I am sorry to tell you, but could you suggest good style of each sentence? I will think more too.

Reply:
@mijin85cho 
I completely agree with you. "Emitted " is not proper.
Can we use the verb "Triggered"?

`View` provides a variety of `events` that are commonly used by the components. The following table lists the basic events provided by `View`:

| Event               | Description                                               |
| ------------------- | --------------------------------------------------------- |
| `KeyEvent`          | Triggered when the key event is received.                   |
| `TouchEvent`        | Triggered when the touch input is received.                 |


When we write "The key event is received.", it seems incomplete. Since it is understood by the reader what is `KeyEvent`, it will be good if we can answer the other two.

@mijin85cho and @bshsqa
Please let me know your valuable feedback on the same.

Reply:
@Reni-90 Yes, "Triggered" is much better than "Emitted". Thanks for good suggestion.

Reply:
I changed it. Thank you for your discussion.